src/core/main.c

```diff
@@ -2736,6 +2736,10 @@ int main(int argc, char *argv[]) {
 
         assert_se(argc > 0 && !isempty(argv[0]));
 
+        /* If we're running in a container, let's immediately try to read logging arguments from our cmdline. */
+        if (getpid_cached() == 1 && detect_container() > 0)
```

Comment:
hmm, container detection is typically not going to work, since /proc is not around here yet in the general case.

Reply:
I dropped this one and only kept the one after we mount /proc